### PR TITLE
Add Icommon icon adding MO

### PR DESCRIPTION
**Pull request by AmauryLiet**
A new MO explaining how to add new icons to your app if you are using Icomoon

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/icomoon-mo/react-native/features/icomoon.mo.md">icomoon.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/features/icomoon.mo.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ 
+ 7. On the bottom of the screen, click `Generate font` then `Download`
+ 
+
+## 2. Add the icons to the application
+
+ 1. Open the `icomoon.zip` file
+ 
+ 2. Overwrite the following files:
+  - `android/app/src/main/assets/fonts/icomoon.ttf`
```
Comment (yleflour) [inline react-native/features/icomoon.mo.md:45]: Why overwrite? Shouldn't we name it based on our appName?

File: react-native/features/icomoon.mo.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - `src/components/Icon/selection.json`
+  - `resources/icons/icomoon.ttf`
+
+
+## 3. [If you are using code-push] Don't forget to hard deploy
```
Comment (yleflour) [inline react-native/features/icomoon.mo.md]: Shouldn't be a 3. Use a quote block with > and the ⚠️ icon

**Review by yleflour — CHANGES_REQUESTED** [review]

File: react-native/features/icomoon.mo.md (hunk 1 above)
Comment (AmauryLiet) [reply react-native/features/icomoon.mo.md:45]: Do you mean creating the file `android/app/src/main/assets/fonts/<APP_NAME>.ttf`?
That's not something I usually do, keeping the default `icomoon.ttf` seems simpler

Why should we rename the ttf file?

Comment (AmauryLiet) [reply react-native/features/icomoon.mo.md:45]: @yleflour what's your mind on this? :)

**Review by tychota — APPROVED** [review]
For me it is ok. Waiting on @yleflour

**Comment by AmauryLiet** [comment]
@yleflour Any feedback on the `icomoon.ttf` file name or are we ok to merge this one?

**Review by tychota — APPROVED** [review]

**Pull request merged**